README.md

```diff
@@ -34,29 +34,32 @@ Add permissions in `AndroidManifest.xml`:
 <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
 ```
 
-
 # API Reference
 
 ## Methods
 
-### `launchCamera()`
+### `launchCamera()/launchCameraAsync()`
```

Comment:
For now lets stick with just `launchCamera()` where the response will come in callback. Two API's is maintenance headache. Users can easily wrap it up in Promise if they want.